README.md

```diff
@@ -67,35 +85,66 @@ CKEDITOR.replace('editor', {
 
 ## Configuration
 
-Initialize a plugin with additional options:
+### Plugin configuration
+
+To apply a custom configuration, initialize the plugin providing additional
+options:
 
 ```javascript
-UPLOADCARE_LOCALE = 'ru' // set locale if you wish
+UPLOADCARE_LOCALE = 'ru' // set a preferred locale if needed
 
 CKEDITOR.replace('editor', {
   extraPlugins: 'uploadcare',
-  toolbar: [['Uploadcare', /* Your toolbar items */]],
+  toolbar: [['Uploadcare', /* your toolbar items */]],
   uploadcare: {
-    publicKey: 'YOUR_PUBLIC_KEY',
-    multiple: true,
-    crop: '1:1,4:3',
+    publicKey: 'YOUR_PUBLIC_KEY', // set your public API key here
+    multiple: true, // allow multi-file uploads
+    crop: '1:1,4:3', // set crop options when handling images
+    /* feel free to add more “object key” options here*/
   },
 })
 ```
 
-You can deeply customize the widget behavior: file sources, file validation, and much more.
-Please, check out the [Uploadcare Widget][widget-docs-config] and [JavaScript API][widget-docs-js-api] docs.
+### Widget configuration
+
+Uploadcare Widget can be deeply customized to suit your UX/UI. You can define
+allowed upload sources, implement file validation, and more.
+
+Use our live [widget sandbox][widget-configure] as a starting point and consider
+checking out the docs on [widget configuration][widget-docs-config] and its
+[JavaScript API][widget-docs-js-api].
+
+## Contributors
+
+* [grayhound](https://github.com/grayhound)
+* [Zmoki](https://github.com/Zmoki)
+* [@dmitry-mukhin](https://github.com/dmitry-mukhin)
+* [nd0ut](https://github.com/nd0ut)
+* [homm](https://github.com/homm)
+* [disolovyov](https://github.com/disolovyov)
+* [zeroasterisk](https://github.com/zeroasterisk)
```

Comment:
Maybe add link to github https://github.com/uploadcare/uploadcare-ckeditor/graphs/contributors ?